build/OpenTelemetryContrib.test.ruleset

```diff
@@ -4,6 +4,36 @@
     <Name Resource="OpenTelemetrySDKRules_Name" />
     <Description Resource="OpenTelemetrySDKRules_Description" />
   </Localization>
+  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
+    <Rule Id="CA1001" Action="None" />
```

Comment:
Question for potential follow ups.
Do you think that on this list there is any other important warning?

Reply:
I don't think there was anything particularly _important_, if I recall correctly.